web_src/js/features/repo-findfile.ts

```diff
@@ -90,6 +90,8 @@ function filterRepoFiles(filter) {
       const span = document.createElement('span');
       // safely escape by using textContent
       span.textContent = part;
+      span.setAttribute('data-tooltip-content', span.textContent);
+      span.setAttribute('data-tooltip-placement', 'top');
```

Comment:
This tooltip migh be annoying. `title` attribute may be less annoying.

Reply:
Oh, yes. I want to change it to this. But I didn't know how. Thanks!